internal/cmd/beta/server/describe/describe.go

```diff
@@ -184,35 +187,50 @@ func outputResult(p *print.Printer, model *inputModel, server *iaas.Server) erro
 			table.AddSeparator()
 		}
 
+		if server.ServiceAccountMails != nil && len(*server.ServiceAccountMails) > 0 {
+			table.AddRow("SERVICE ACCOUNTS", strings.Join(*server.ServiceAccountMails, "\n"))
+			table.AddSeparator()
+		}
+
 		if server.Volumes != nil && len(*server.Volumes) > 0 {
 			volumes := []string{}
 			volumes = append(volumes, *server.Volumes...)
 			table.AddRow("VOLUMES", strings.Join(volumes, "\n"))
 			table.AddSeparator()
 		}
 
-		if model.Details {
-			if server.ServiceAccountMails != nil && len(*server.ServiceAccountMails) > 0 {
-				emails := []string{}
-				emails = append(emails, *server.ServiceAccountMails...)
-				table.AddRow("SERVICE ACCOUNTS", strings.Join(emails, "\n"))
-				table.AddSeparator()
-			}
+		err := table.Display(p)
+		if err != nil {
+			return fmt.Errorf("render table: %w", err)
+		}
+
+		// If no --details is set, we only display the overview table
+		if !model.Details {
```

Comment:
aren't there more tables to add besides the attached nics, like the public IP for example? 
If not, I'm not sure why we need to add this "--details" flag instead of just adding the nic row to the existing table

Reply:
Ah, ok I agree. Because if they don't know the existence of the flag would never see the NICs (when using `pretty` output)

Reply:
WIll remove it then